### PR TITLE
[N/A] Authentication screen

### PR DESCRIPTION
**Pull request by pedrofelipe**
This PR creates a authentication screen and also defines a architecture to Sass files.
- [x] Desktop
- [x] Tablet/Mobile adjustments

screen shot 2016-10-07 at 2 40 26 pm

screen shot 2016-10-07 at 4 06 37 pm

screen shot 2016-10-07 at 10 52 30 am

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "is-queue",
+  "name": "is-firebase-queue",
```
Comment (gabrielrinaldi) [inline bower.json]: I would name it firebase-queue-ui instead.

**Review by gabrielrinaldi — CHANGES_REQUESTED** [review]
Can we use the same color scheme we see here:
screen shot 2016-10-07 at 9 54 37 am

**Comment by gabrielrinaldi** [comment]
Also, please don't use our logo, if you are going to use a logo use Firebase's

**Comment by pedrofelipe** [comment]
@gabrielrinaldi updated. would you mind to take a look again?

**Review by viniciusalmeida — APPROVED** [review]
👍 
Can you add a screen shot with the mobile resolution to PR's description?

**Comment by pedrofelipe** [comment]
@viniciusalmeida done! ;)

**Pull request merged**